_topics/en/mast.md

```diff
@@ -0,0 +1,82 @@
+---
+title: MAST
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Soft Forks
+  - Scripts and Addresses
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **MAST** is a method of using a merkle tree to store the various
+  user-selected conditions that must be fulfilled in order for the
+  encumbered bitcoins to be spent.  Using a merkle tree allows the
+  spender to select which one of the conditions they'll fulfill without
+  having to reveal the details of other conditions to the block chain.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Users of MAST who are able to keep unused conditions off of the block
+  chain will enjoy lower fees, be able to create larger contracts than
+  currently possible, will have improved privacy, and will improve the
+  fungibility of their bitcoins.
```

Comment:
I think this is correct as-is: "Users of MAST who are able to keep unused conditions off of the block chain [...] improve the fungibility of their bitcoins."

- For "of everyone's bitcoins" I think this is incorrect because more people using MAST actually reduces the fungibility of coins that don't use MAST in the same way that someone who hasn't updated their wallet in years and is still using uncompressed pubkeys probably makes their transactions much easier to identify and censor.

- For "improve the fungibility of bitcoin" I don't think that phrase makes sense.  Compare it to "improve the fungibility of dollar".  Maybe you mean big-B Bitcoin?